packages/view/src/Charcoal/View/ViewServiceProvider.php

```diff
@@ -298,6 +307,57 @@ protected function registerTwigTemplatingServices(Container $container)
         };
     }
 
+    /**
+     * @param Container $container The DI container.
+     * @return void
+     */
+    protected function registerTwigHelpersServices(Container $container)
+    {
+        if (!isset($container['view/twig/helpers'])) {
+            $container['view/twig/helpers'] = function () {
+                return [];
+            };
+        }
+
+        /**
+         * Translation helpers for Twig.
+         *
+         * @return TranslatorHelpers
+         */
```

Comment:
Remove incorrect block comments or fix return type. Same for other services declared afterwards.